README.md

```diff
@@ -48,3 +52,5 @@ Have a patch that will benefit this project? Awesome! Follow these steps to have
 
 ## License
 All files in this repository are under the [MIT License](LICENSE) unless noted otherwise.
+
+[0]: https://cloud.google.com/appengine/docs/php/googlestorage/setup
```

Comment:
this way keeps lines from extending over 80 characters long, and also it's something we do consistently in other repositories.